### DISCUSSION TITLE
Add eraser of selected area to Inpainter interface.

### DISCUSSION THREAD

**Discussion by AnalyticsXXI** [lllyasviel/Fooocus#2954] [Ideas]
After carefully selecting some area to modify, one may decide to reduce it area or accidentally select some area which shouldn't be there. Currently, it require to start selection all over again. With tool like eraser, it would be so easier, and it would be less intense to select the area, if can easily change it any time you want.

**Comment by wisdombox44** [comment]
Control+Z

**Comment by AnalyticsXXI** [comment]
> Control+Z

First, Control+Z doesn't work as we would expect, it reverses not the last painting, but some overall painting over previous time, like smoothing few actions and cancel each a little bit. 

Second, it would not be flexible anyway. The point is to be able to change selected area, not only increase it or cancel last action.
